qa/integration/services/filebeat_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+current_dir="$(dirname "$0")"
+
+if [ -n "${FILEBEAT_VERSION}" ]; then
+  echo "Filebeat version is $FILEBEAT_VERSION"
+  version=$FILEBEAT_VERSION
+else
+   version=5.0.0-alpha5
```

Comment:
if possible I suggest picking this version from the versions.yml file, so we don't have to maintain this :)